code/modules/clothing/head/collectable.dm

```diff
@@ -4,6 +4,7 @@
 /obj/item/clothing/head/collectable
 	name = "collectable hat"
 	desc = "A rare collectable hat."
+	icon_state = "viking"
```

Comment:
this should fallback into the default "no-texture" texture

Reply:
It spawns on raiders sometimes as an invisible hat, and it's annoying. Either this or removing it from the spawn chance.